src/util/nestedArray/nestedArray.ts

```diff
@@ -0,0 +1,22 @@
+export function nestedIndexOf(hay: any[][], needle: any[]): number {
+	let i = 0;
+
+	hay.forEach((e: any) => {
```

Comment:
you can add a second parameter to this function that will be the index: `(item: any, index: number)`.